### ISSUE TITLE
how to reconstruct my own pointcloud?

### ISSUE THREAD

**Issue by YuanYe-99** [ErlerPhilipp/points2surf#28]
Thanks for your excellent work! I have a problem with your code. After using make_pc_dataset.py I don't know how to reconstruct the point cloud to a mesh. Looking forward to your reply.

**Comment by ErlerPhilipp** [comment]
Right, I didn't write it explicitly in the readme. The best solution is most likely to duplicate a matching experiment eval script, e.g. experiments/eval_p2s_max.sh, and adapt it to your new dataset. I recommend picking the one for the max model.

In this file, you need to specify your testset.txt in the dataset parameter. In case you have strong noise, you may also need to increase the epsilon, certainty_threshold, and sigma parameters.

**Comment by YuanYe-99** [comment]
> Right, I didn't write it explicitly in the readme. The best solution is most likely to duplicate a matching experiment eval script, e.g. experiments/eval_p2s_max.sh, and adapt it to your new dataset. I recommend picking the one for the max model.
> 
> In this file, you need to specify your testset.txt in the dataset parameter. In case you have strong noise, you may also need to increase the epsilon, certainty_threshold, and sigma parameters.

Thanks for your reply! I will try later.

**Comment by ErlerPhilipp** [comment]
are the files and paths really correct?
you should see several folders in datasets/our.
make sure to call the experiment scripts from the repo root like `bash experiments/eval_p2s_max.sh`

**Comment by ErlerPhilipp** [comment]
your dataset contains only point clouds, right? so there are no ground-truth meshes? this means that parts of the evaluation don't make sense anyway. you can just ignore them as they will only try to produce tables for e.g. chamfer distance.

you should find the reconstructions in a folder like results/max_model/own.

**Comment by YuanYe-99** [comment]
> your dataset contains only point clouds, right? so there are no ground-truth meshes? this means that parts of the evaluation don't make sense anyway. you can just ignore them as they will only try to produce tables for e.g. chamfer distance.
> 
> you should find the reconstructions in a folder like results/max_model/own.

I'm truly grateful for your help.

**Comment by ErlerPhilipp** [comment]
i'm happy if this is useful for you. if there are any other issues, feel free to re-open this issue or create a new one.
